frontend/ui/widget/qrwidget.lua

```diff
@@ -29,8 +29,8 @@ function QRWidget:init()
         local grid_size = scale * #grid
         local bb = Blitbuffer.new(grid_size, grid_size)
         local white = Blitbuffer.COLOR_WHITE
-        for x, col in ipairs(grid) do
-            for y, lgn in ipairs(col) do
+        for x, col in pairs(grid) do
```

Comment:
If you use #grid, it's an array. I don't think pairs() would be more efficient that ipairs() - but ipairs() is clearer (and won't have us wondering whether this is a bug in a few years :)

Reply:
Sorry, I was misinformed and thought that ipairs was slower than pairs.

Reply:
(I really don't know if it is slower or not - it just feels wrong to use pairs on an array.)